packages/nux/src/components/inline-tip/index.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+import { Notice } from '@wordpress/components';
+import { compose } from '@wordpress/compose';
+import { withSelect, withDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+
+export function InlineTip( { isVisible, className, onRemove, children } ) {
+	if ( ! isVisible ) {
+		return null;
+	}
+
+	return (
+		<Notice className={ className } onRemove={ onRemove }>
```

Comment:
For `onRemove`, it looks like the `Notice` component provides the label 'Dismiss this notice'. For tips it looks like it allows the user to remove all tips. I think it'd be good to show a more accurate label.

Reply:
Do you think it makes sense to make the label say just _Dismiss_? Or to make the label customisable via a prop? I'm on the fence. I fear the latter might make the API of the `Notice` component too wishy washy<sup>1</sup>.

<sup>1</sup> Highly technical term.

Reply:
I'm on the fence too. 'Dismiss' sounds good to me and seems clear. Go with whatever you think is best.

Reply:
I think let's stick with 'Dismiss this notice'. The word 'notice' is already pretty vague and I noticed we've used this copy for other non-notices e.g. snackbars.